src/base1/Makefile.am

```diff
@@ -23,6 +23,8 @@ basedebug_DATA = \
 	dist/base1/cockpit.min.js.map \
 	$(NULL)
 
+AM_TESTS_ENVIRONMENT = export G_DEBUG=fatal-criticals ;
```

Comment:
Okay fair enough. I'm fine doing that as a follow up. 

If the warnings are triggered from ```test-xxx``` binaries they can be covered with ```cockpit_expect_warning()``` and similar.